ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/individual-journey-routing.module.ts

```diff
@@ -21,24 +21,25 @@ import { AccessToRoomComponent } from './pages/access-to-room/access-to-room.com
 import { ConsentComponent } from './pages/consent/consent.component';
 import { ThankYouComponent } from './pages/thank-you/thank-you.component';
 import { VideoViewComponent } from './components/video-view/video-view.component';
+import { IndividualGuard } from './individual.gaurd';
 
 const routes: Routes = [
-  { path: Paths.AboutHearings, component: AboutHearingsComponent },
-  { path: Paths.DifferentHearingTypes, component: DifferentHearingTypesComponent },
-  { path: Paths.ExploreCourtBuilding, component: ExploreCourtBuildingComponent },
-  { path: Paths.CourtBuildingVideo, component: CourtBuildingVideoComponent },
-  { path: Paths.ExploreVideoHearing, component: ExploreVideoHearingComponent },
-  { path: Paths.UseCameraMicrophone, component: UseCameraMicrophoneComponent },
-  { path: Paths.ParticipantView, component: ParticipantViewComponent },
-  { path: Paths.HelpTheCourtDecide, component: HelpTheCourtDecideComponent },
-  { path: Paths.AboutYou, component: AboutYouComponent },
-  { path: Paths.Interpreter, component: InterpreterComponent },
-  { path: Paths.YourComputer, component: YourComputerComponent },
-  { path: Paths.AboutYourComputer, component: AboutYourComputerComponent },
-  { path: Paths.YourInternetConnection, component: YourInternetConnectionComponent },
-  { path: Paths.AccessToRoom, component: AccessToRoomComponent },
-  { path: Paths.Consent, component: ConsentComponent },
-  { path: Paths.ThankYou, component: ThankYouComponent }
+  { path: Paths.AboutHearings, component: AboutHearingsComponent, canActivate: [IndividualGuard] },
```

Comment:
Will this call the userService for every route ? If so it will slow down every page load.. ???